lib/GetStream/StreamChat/Channel.php

```diff
@@ -432,14 +432,6 @@ public function deleteImage(string $url): StreamResponse
         return $this->client->delete($this->getUrl() . '/image', ["url" => $url]);
     }
 
-    /** @link https://getstream.io/chat/docs/php/custom_events/?language=php
-      * @throws StreamException
-      */
-    public function sendCustomEvent(array $event): StreamResponse
```

Comment:
ok, this is wrong but what are we doing about channel one?

Reply:
yeah the other (user event) is in `Client.php`

https://github.com/GetStream/stream-chat-php/pull/80/files#diff-7f302ac38049dc9f1a8c5b3a8a13b7d99cb8dd5ae99ceadf8adaf6799323d2f5

Reply:
yes but why are we removing this one?

Reply:
it was duplicated (my mistake 🤦). there is a other method in Channel.php already.

Reply:
I see, `sendEvent` is there. Good to go then